main.js

```diff
@@ -157,7 +158,8 @@ app.get('/rdm', (request, response) => {
     }
 
     if (modelId in register.rdm[manufacturerId].models) {
-      response.redirect(301, `/${manufacturer.key}/${manufacturer.models[modelId]}`);
+      const hash = (personalityIndex === undefined || personalityIndex === '') ? '' : `#rdm-personality-${personalityIndex}`;
```

Comment:
It looks like hash may be a keyword/reserved word, and I wouldn't be surprised if it was...